src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -145,7 +145,7 @@ static TDSType valueOf(int intValue) throws IllegalArgumentException {
     DECIMAL        (Category.NUMERIC,         "decimal",          JDBCType.DECIMAL),
     NUMERIC        (Category.NUMERIC,         "numeric",          JDBCType.NUMERIC),
     GUID           (Category.GUID,            "uniqueidentifier", JDBCType.GUID),
-    SQL_VARIANT    (Category.VARIANT,         "sql_variant",      JDBCType.VARCHAR),
+    SQL_VARIANT    (Category.SQL_VARIANT,     "sql_variant",      JDBCType.VARCHAR),  
```

Comment:
thanks for explanation 👍 

One more question, why is SQL_VARIANT mapped to JDBCType.VARCHAR? should it be JDBCType.SQL_VARIANT?